src/pe_reports/metrics.py

```diff
@@ -12,25 +22,571 @@ def __init__(self, start_date, end_date, org_uid):
+
+        breach_df.columns = breach_df.columns.droplevel(1)
+        breach_df = breach_df.rename(columns={"email": "number_of_creds"})
+        breach_df = breach_df[
+            [
+                "breach_name",
+                "breach_date",
+                "modified_date",
+                "password_included",
+                "number_of_creds",
+            ]
+        ]
+        breach_det_df = breach_df.rename(columns={"modified_date": "update_date"})
+
+        if len(breach_det_df) > 0:
+            breach_det_df["update_date"] = breach_det_df["update_date"].dt.strftime(
+                "%m/%d/%y"
+            )
+            breach_det_df["breach_date"] = pd.to_datetime(
+                breach_det_df["breach_date"]
+            ).dt.strftime("%m/%d/%y")
+
+        breach_det_df = breach_det_df.rename(
+            columns={
+                "breach_name": "Breach Name",
+                "breach_date": "Breach Date",
+                "update_date": "Date Reported",
+                "password_included": "Password Included",
+                "number_of_creds": "Number of Creds",
+            }
+        )
+
+        return breach_det_df
+
+    def password(self):
+        """Return total number of credentials with passwords."""
+        pw_creds_csg = len(
+            self.query_cyberSix_creds[self.query_cyberSix_creds["password_included"]]
+        )
+        pw_creds_hibp = len(
+            self.query_hibp_view[self.query_hibp_view["password_included"]]
+        )
+        return pw_creds_csg + pw_creds_hibp
+
     def total(self):
-        """Return total number of credentials."""
+        """Return total number of credentials found in breaches."""
         df_cred_csg = self.query_cyberSix_creds.shape[0]
         df_cred_hibp = self.query_hibp_view.shape[0]
         return df_cred_csg + df_cred_hibp
 
-    # TODO the following functions correspond to functions at report_generator.py
-    # TODO and will be added in follow up PR found here: https://github.com/cisagov/pe-reports/pull/99
-    def password(self):
-        """Return total number of credentials with passwords."""
 
-    def breached(self):
-        """Return total number of breached credentials."""
+class Domains_Masqs:
+    """Domains Masquerading Class."""
 
-    def by_days(self):
-        """Return number of credentials by day."""
+    def __init__(self, start_date, end_date, org_uid):
+        """Initialize domains masquerading class."""
+        self.start_date = start_date
+        self.end_date = end_date
+        self.org_uid = org_uid
+        df = query_domMasq(org_uid, start_date, end_date)
+        self.df_mal = df[df["malicious"]]
 
-    def breach_details(self):
-        """Return breach details."""
+    def count(self):
+        """Return total count of malicious domains."""
+        df = self.df_mal
+        return len(df.index)
+
+    def summary(self):
+        """Return domain masquerading summary information."""
+        df_mal = self.df_mal
```

Comment:
This line seems unnecessary to me- you can just use `self.df_mal` throughout this function.

Reply:
@dav3r, I've addressed this comment in the following commit: [cc99f8b](https://github.com/cisagov/pe-reports/pull/99/commits/cc99f8b7733cb76a25af69a6b49eee2b7cfd9bc9)